src/main/resources/db/migration/changes/008-create-Admins.json

```
@@ -0,0 +1,43 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "008-create-Admins",
+        "author": "ShellyZ",
```

```suggestion
        "author": "shellyzhu",
```

Name should match the github id